cfgov/unprocessed/css/pages/home.less

```diff
@@ -0,0 +1,106 @@
+.content_main__home {
+    padding-top: unit( @grid_gutter-width / @base-font-size-px, em );
+
+    .respond-to-max( @bp-xs-max, {
+        padding-bottom: unit( ( @grid_gutter-width * 2 ) / @base-font-size-px, em );
+    } );
+}
+
+.content-l_col__home {
+    + .content-l_col__home {
+        .respond-to-max( @bp-med-max, {
+            margin-top: unit( ( @grid_gutter-width ) / @base-font-size-px, em );
+        } );
+
+        .respond-to-range( @bp-sm-min, @bp-med-max, {
+            &:before {
+                .a-rule-break();
+                content: '';
```

Comment:
I'm not sure why, but I've seen `'\a0'` used to fill `content` with empty content. There may be a cross-browser reason. Not sure. Perhaps worth looking into at some point.